ci/trace-requirements.sh

```diff
@@ -4,22 +4,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-oft_version="3.5.0"
+oft_version="3.6.0"
 tmp_dir="/tmp/oft/"
-tmp_file="$tmp_dir/openfasttrace-$oft_version.jar"
+jar_file="$tmp_dir/openfasttrace-$oft_version.jar"
 
 base_dir="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
 readonly base_dir
 
-if [[ ! -f "$tmp_file" ]]; then
+if [[ ! -f "$jar_file" ]]; then
     mkdir -p "$tmp_dir"
     url="https://repo1.maven.org/maven2/org/itsallcode/openfasttrace/openfasttrace/$oft_version/openfasttrace-$oft_version.jar"
-    echo "Downloading $url to $tmp_file"
-    curl --output "$tmp_file" "$url"
+    echo "Downloading $url to $jar_file"
+    curl --output "$jar_file" "$url"
```

Comment:
Add checksum check please.

Reply:
I am using maven now to download the jar. This automatically verifies the checksum.